components/home/whoWeAre.tsx

```diff
@@ -196,6 +197,32 @@ export default function WhoWeAre() {
               linkedIn="https://www.linkedin.com/in/mariasilverhardt"
             />
           </Col>
+
+          <Col md={4} className="d-flex justify-content-center">
+            <TeamMember
+              name="Krishna Glick"
+              image="krishna_glick.jpg"
+              title="Senior Software Engineer"
+              description={
+                <span>
+                  🥦🧗🏻🐶🐱
+                  <br /> Previously:
+                  <br />
+                  Senior Software Engineer{" "}
+                  <a
+                    href="https://www.iheart.com"
+                    target="_blank"
+                    rel="noreferrer nofollow"
+                  >
+                    @iHeartMedia
+                  </a>
+                </span>
+              }
+              github="https://github.com/krishnaglick"
+              linkedIn="https://twitter.com/msdatadetective"
+              twitter=""
```

Comment:
Should I make a twitter for myself as a grouparoo employee?

Reply:
If you don't have a Twitter account, no need to make a new one.  In that case, the component should be updated to handle a "" or null twitter account to hide the icon